python/dolfinx/wrappers/common.cpp

```diff
@@ -145,16 +161,32 @@ void common(nb::module_& m)
             return std::pair(std::move(map),
                              dolfinx_wrappers::as_nbarray(std::move(ghosts)));
           },
-          nb::arg("entities"));
+          nb::arg("entities"),
+          nb::raw_doc("Create new index map from a subset of indices in this "
```

Comment:
This is too fragile. The approach we've adopted (which is not fully complete) is to document from Python.

Reply:
So then we need to wrap every function we expose to the Python interface?